### DISCUSSION TITLE
SEBE Visualisation

### DISCUSSION THREAD

**Discussion by FPYS** [UMEP-dev/UMEP#303] [Q&A]
Hi,

I've got the following error trying to visualise SEBE results : 

RuntimeError: C:/Users/Florian/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual/data/temp.tif: No such file or directory

My files : 
[SEBE.zip](https://github.com/UMEP-dev/UMEP/files/6739998/SEBE.zip)

**Comment by biglimp** [comment]
I tried with your files and it worked just fine. What version of UMEP and QGIS are you using? Below is an example from the tool using your data.

![image](https://user-images.githubusercontent.com/11908356/123963448-c0623600-d9b2-11eb-807b-af328d5af5bf.png)
